artifacts/Products/ShowProducts.recipes

```diff
@@ -13,17 +13,25 @@ particle ShowProduct in 'source/ShowProduct.js'
   in Product product
   consume item
 
-recipe MuxedProductItem &muxProduct
+recipe MuxedItem &muxItem
   ItemMultiplexer
 
+// TODO(mmandlis): List + &muxItem should be a generic recipe used by both -
+// show products and show wishlist recipes. 
 recipe ProductList &showShopList
-  copy #shoplist as newlist
+  copy #shoplist as shoplist
   List
-    items = newlist
+    items <- shoplist
+  &muxItem
```

Comment:
Again, my bad since I mixed up which file I was looking at. However, I am anxious to get rid of some of this muxing/list boilerplate, and it seems to be growing here. I'll have to study it more to understand exactly why.

Reply:
We never reached that moment. The multiplexer is still declared in the recipe muxProduct/muxItem. The fact that we can automatically aggregate it in the prior case without explicit reference doesn't mean that the untidy reference to multiplexing has actually gone away.

In fact, there are sorta kinda *2* multiplexers here - even though List pretends that it doesn't do multiplexing it is in fact acting as a multiplexer by generating the set slot.

In my view, we should eventually be able to automatically coalesce a world where a list of things (newlist/shoplist) can be applied to a singleton particle (ShowProduct?) without any recipes referencing either List _or_ ItemMultiplexer.

My challenge to both of you is to figure out why that doesn't work, and either fix it or let me know so that I can :)